website/versioned_sidebars/version-0.15.0-sidebars.json

```diff
@@ -4,5 +4,11 @@
       "type": "autogenerated",
       "dirName": "."
     }
+  ],
+  "docs": [
+    {
+      "type": "autogenerated",
+      "dirName": "."
```

Comment:
I have no idea whether this is strictly necessary, but my local build was unhappy if I clicked on certain pages and this was not included so perhaps the real one will be too? Will I need to backfill all the others?